apps/docs/app/lib/getComponentProps.ts

```diff
@@ -42,6 +42,7 @@ export async function getComponentProps(component: string) {
     const [item] = data;
 
```

Comment:
![image](https://github.com/user-attachments/assets/f37bc996-8c01-46a7-af25-21b32ccb4535)
This could be fixed in another PR but Listbox I don't think the absence of spacing between the divider and the items are per design.